package.json

```diff
@@ -63,7 +65,7 @@
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true craco start",
     "build": "SKIP_PREFLIGHT_CHECK=true craco build",
-    "test": "SKIP_PREFLIGHT_CHECK=true craco test --watchAll",
+    "test": "SKIP_PREFLIGHT_CHECK=true craco test --env=jest-environment-jsdom-sixteen --watchAll",
```

Comment:
this is a workaround for new version of testing library, will replace when they have a solution

Reply:
please add a todo comment here.

Reply:
I believe unfortunately we can't add  comments in JSON, where else do you think adding a comment for this would be meaningful?

Reply:
its ok, just keep in mind we have to update it